index.html

```diff
@@ -40,53 +45,50 @@ <h3>LET'S CONNECT</h3>
                 </div>
             </footer>
         </section>
+      </section>
       <section id="my_work">
         <section id="p1">
-          <div id="introd_1">
-                <img  class="project_image" src="statics/images/Snapshoot_Portfolio.png" alt="snapshoot image project">
-                <h1>Tonic</h1>
-                <div id="general_inf">
-                    <p class="title">CANOPY</p>
-                    <img class="counter" src="statics/images/Counter.png" alt="counter"/>
-                    <p class="job-title">Back End Dev</p>
-                    <img class="counter" src="statics/images/Counter.png" alt="counter">
-                    <p class="year">2015</p>
-                </div>
-
-               <p class="project_description">
-                A daily selection of privately personalized reads; no accounts or sign-ups required.
-               </p>
-          </div>
-          <footer>
-             <ul id="skills_0" class="skills">
-               <li>html</li>
-               <li>css</li>
-               <li>javaScript</li>
-             </ul>
-             <div class="external_link">
-                 <a href="#">See Project</a>
-             </div>
-            </footer>
-
-
+          <img  class="project_image" src="statics/images/Snapshoot_Portfolio.png" alt="snapshoot image project">
+          <div class="general">
+              <h3>Tonic</h3>
+              <div id="general_inf">
+                 <p class="title">CANOPY</p>
+                 <img class="counter" src="statics/images/Counter.png" alt="counter"/>
+                 <p class="job-title">Back End Dev</p>
+                 <img class="counter" src="statics/images/Counter.png" alt="counter">
+                 <p class="year">2015</p>
+              </div>
+              <p class="project_description">
+                      A daily selection of privately personalized reads; no accounts or sign-ups required.
+              </p>
+              <footer>
+                  <ul id="skills_0" class="skills">
+                     <li>html</li>
+                     <li>css</li>
+                     <li>javaScript</li>
+                  </ul>
+                  <div class="external_link">
+                     <a href="#">See Project</a>
```

Comment:
- [OPTIONAL] For each `See Project` button the text color change to white on hover.